ecs.conf

```diff
@@ -0,0 +1,21 @@
+# Copyright 2014-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I suspect that this file was accidentally reintroduced, probably by the original PR by @samuelkarp. In any case, there shouldn't be a top-level `ecs.conf` in the repo.